dist/pythonlibs/riotctrl_shell/tests/test_gnrc.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (C) 2020 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import logging
+import os
+import riotctrl_shell.gnrc
+import subprocess
+
+from common import init_ctrl
+
+
+logger = logging.getLogger(__name__)
+
+
+def tap_exists(tap="tap0"):
```

Comment:
Do you think we could get rid of calling `ip link show tap` ? That would simplify automatic testing.

Reply:
Oops I planned to remove this but forgot ^^"

Reply:
Removed.